upgrade/install-3.0.php

```diff
@@ -79,5 +79,15 @@ function upgrade_module_3_0($object, $install = false)
 			) ENGINE='._MYSQL_ENGINE_.' DEFAULT CHARSET=utf8 AUTO_INCREMENT=1');
 	}
 
+	//Add datas to paypal order
+	Db::getInstance()->Execute('
```

Comment:
If a merchant is on 3.8.2 : 
- If user installed module on 3.8.2 then table has been installed with the correct columns so no problem. 
- If user upgraded his paypal from a 2.8.0 to a 3.8.2 => Then user had an error and he had to come to support to handle the error. 

So in all cases people who are today in 3.8.2 have an error that is blocking and so they contacted us, or they've installed the right version with correct columns. This is why I would prefer not to alter the table on 3.8.3.